flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/NettyClientServerSslTest.java

```diff
@@ -98,21 +86,48 @@ public void testValidSslConnectionAdvanced() throws Exception {
 		Channel ch = NettyTestUtil.connect(serverAndClient);
 
 		SslHandler sslHandler = (SslHandler) ch.pipeline().get("ssl");
-		assertEquals(sslHandler.getHandshakeTimeoutMillis(), handshakeTimeout);
-		assertEquals(sslHandler.getCloseNotifyTimeoutMillis(), closeNotifyFlushTimeout);
+		int handshakeTimeout = sslConfig.getInteger(SSL_HANDSHAKE_TIMEOUT);
```

Comment:
```
assertSslConfig(sslConfig.getInteger(SSL_HANDSHAKE_TIMEOUT), sslHandler.getHandshakeTimeoutMillis())
```
and do:
```
assertSslConfig(expected, actual) {
  if (expected != -1) {
    assertEquals(expected, actual)
  }
  else {
    assertTrue(...);
  }
}
```
in 4 places here? Maybe renaming it to `assertEqualsOrMinusAsDefaultValue`?